### PR TITLE
Support Namespaced Statuses

### PR DESCRIPTION
**Pull request by sam-heilbron**
Blocked by: solo-kit release (to update https://github.com/solo-io/solo-apis/pull/226/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R33)

### CODE REVIEW

File: test/scripts/expected_skv2_result.test [hunk 1]
```diff
@@ -264,3 +264,9 @@ message VirtualServiceStatus {
 	// Opaque details about status results
```
Comment (kdorosh) [inline test/scripts/expected_skv2_result.test:264]: do we want to be committing this .test file?

Comment (sam-heilbron) [reply test/scripts/expected_skv2_result.test:264]: this existed before. It was a test mostly used to confirm that skv2 codegen doesn't change underneath us. My changes did change it, so the test result had to be updated as well

File: api/gloo/enterprise.gloo/v1/auth_config.proto [hunk 2]
```diff
@@ -23,6 +23,7 @@ import "google/protobuf/empty.proto";
 message AuthConfigSpec {
 
 
+    reserved 1;
```
Comment (MitchAman) [inline api/gloo/enterprise.gloo/v1/auth_config.proto:26]: Here and other `*.proto` files in this PR have a reserved field added, but no field removed. Why is this necessary?

**Comment by sam-heilbron** [comment]
Closed in favor of: https://github.com/solo-io/solo-apis/pull/229
